cli/src/commands/import/csv.ts

```diff
@@ -38,21 +37,14 @@ export default class ImportCSV extends BaseCommand<typeof ImportCSV> {
     create: Flags.boolean({
       description: "Whether the table or columns should be created if they don't exist without asking"
     }),
-    'no-column-name-normalization': Flags.boolean({
-      description: 'Avoid changing column names in a normalized way'
-    }),
     'batch-size': Flags.integer({
       description: 'Batch size to process and upload records'
     }),
     'max-rows': Flags.integer({
       description: 'Maximum number of rows to process'
     }),
-    'skip-rows': Flags.integer({
-      description: 'Number of rows to skip'
-    }),
     delimiter: Flags.string({
-      description: 'Delimiters to use for splitting CSV data',
-      multiple: true
+      description: 'Delimiter to use for splitting CSV data'
```

Comment:
I understand this was made to simplify the input options of the parser, however since papaparse supports multiple delimiters to guess, I think we should keep it multiple here too.

The aim of the multiple option is not that a file has different delimiters on it, but that you accept inputs with different delimiters and you're not certain which one was used and you pass a priority ordered array.

Reply:
The importer package now supports `delimiter` and `delimitersToGuess` which both map 1:1 with papaparse.

<img width="1245" alt="Screenshot 2023-07-06 at 09 52 39" src="https://github.com/xataio/client-ts/assets/6640874/ec4e0f8c-1047-486b-94d4-d356c7b45329">

^^^ papaparse delimiter option. I believe it's a single delimiter, whilst `delimitersToGuess` is a `string[]`

Since this flag was called `delimiter` I just passed it through to `delimiter`. 

We have sensible options for `delimitersToGuess`, it's possible to add a flag for this in future as well.

Reply:
Interesting, yeah then it makes sense to expose the `delimitersToGuess` option too. I assumed we weren't forcing the delimiter and leaving it in auto with `delimiter` -> `delimitersToGuess`

Reply:
Added the flag 👍 : https://github.com/xataio/client-ts/pull/1051/commits/51c254877b2b4767520ad7441601edab92c3eefb